Gemfile

```diff
@@ -36,6 +36,7 @@ group :secondary do
   gem 'asciidoctor', '>= 0.1.0'
   gem 'liquid'
   gem 'maruku'
+  gem 'pandoc-ruby'
```

Comment:
Is this okay to you? Or is there a way to only pandoc functionality when the user has `pandoc-ruby` installed already?

The problem then is: how to run the tests? I don't see a `:test` group in your Gemfile... I think it's not good to load so many gems that aren't needed (I don't need to have different markdown engines available, I always only use one).